ffig/FFIG.py

```diff
@@ -66,15 +66,16 @@ def get_class_name(header_path):
     return re.sub(".h$", "", header_name)
 
 
-def write_bindings_to_disk(module_name, api_classes, env, output_dir):
+def write_bindings_to_disk(args, api_classes, env, output_dir):
     """
     Write the bindings to disk, return Nothing
     Input:
+    - args from command line
     - api_classes
     - environment to get templates from
-    - args
     - output_dir where to write to
     """
+    module_name = args.module_name
     for binding in args.bindings:
```

Comment:
Ugh; because `args` was defined inside an `if __name__ == '__main__'` rather than inside a function.

Ok we should change that, but not by passing `args`. It should pass `module_name` (a string) and `bindings` (a list of strings). Jon and I made some changes last week to avoid passing the whole of `args` around as some big opaque configuration blob; we must have missed this because `args` was a name in the global scope.

It would be good to (separately) fix up `main()` so that all of the argument parsing happens in there and the global scope just calls `main` if `__name__ == '__main__'`.